File: QuickStart/ChartView/Linear/SalesRevenue.vb
Imports System

Namespace Telerik.Examples.WinControls.ChartView.Linear
	Public Class SalesRevenue
		Private month_Renamed As String
		Private revenue_Renamed As Double

		Public Sub New(ByVal month As String, ByVal revenue As Double)
			Me.month_Renamed = month
			Me.revenue_Renamed = revenue
		End Sub

		Public ReadOnly Property Month() As String
			Get
				Return Me.month_Renamed
			End Get
		End Property

		Public ReadOnly Property Revenue() As Double
			Get
				Return Me.revenue_Renamed
			End Get
		End Property
	End Class
End Namespace

File: QuickStart/ChartView/Linear/SalesRevenue.cs
using System;

namespace Telerik.Examples.WinControls.ChartView.Linear
{
    public class SalesRevenue
    {
        private string month;
        private double revenue;

        public SalesRevenue(string month, double revenue)
        {
            this.month = month;
            this.revenue = revenue;
        }

        public string Month
        {
            get
            {
                return this.month;
            }
        }

        public double Revenue
        {
            get
            {
                return this.revenue;
            }
        }
    }
}
